GitCommands/Settings/AppSettings.cs

```diff
@@ -368,6 +368,12 @@ public static int CommitDialogNumberOfPreviousMessages
             set => SetInt("commitDialogNumberOfPreviousMessages", value);
         }
 
+        public static bool CommitDialogShowOnlyMyMessages
+        {
+            get => GetBool("commitDialogShowOnlyMyMessages", false);
```

Comment:
I prefer a default value of 'true'.
what do you all think about it?

Reply:
I prefer false, to not hide messages by default